mesonbuild/compilers/compilers.py

```diff
@@ -83,6 +83,8 @@
     clink_suffixes += lang_suffixes[_l]
 clink_suffixes += ('h', 'll', 's')
 all_suffixes = set(itertools.chain(*lang_suffixes.values(), clink_suffixes))  # type: T.Set[str]
+suffix_to_lang = dict(itertools.chain(*(
+    [(suffix, lang) for suffix in v] for lang, v in lang_suffixes.items()))) # type: T.Dict[str, str]
```

Comment:
this is a module constant, so let's treat it like one (the existing code is old and hasn't been updated). Convention in python is that constants are all caps with underscores:
```python
SUFFIX_TO_LANG: T.Dict[str, str] = ...
```

Reply:
I didn't use the "a: b = c" form for consistency with the other lines, but uppercase is better indeed.